tests/unit/utils/sparqlhandler/test_data_cube_state.py

```diff
@@ -126,19 +125,26 @@ def test_get_data_set_dsd_csv_url_for_csv_url():
     """
     Ensures that the valid data_set_dsd_and_csv_url_for_csv_url property is returned.
     """
+
     csvw_metadata_json_path = (
         _test_case_base_dir
         / "pivoted-single-measure-dataset"
         / "qb-id-10004.csv-metadata.json"
     )
-    csvw_rdf_manager = CsvwRdfManager(csvw_metadata_json_path)
-    data_cube_state = DataCubeState(csvw_rdf_manager.csvw_state)
+
+    csvw_rdf_manager = get_csvw_rdf_manager(csvw_metadata_json_path)
+
+    data_cube_state = get_data_cube_inspector(csvw_metadata_json_path)
+
     primary_catalog_metadata = (
         csvw_rdf_manager.csvw_state.get_primary_catalog_metadata()
     )
 
     data_set_uri = primary_catalog_metadata.dataset_uri
-    data_set_uri = to_absolute_rdflib_file_path(data_set_uri, csvw_metadata_json_path)
+    data_set_uri = to_absolute_rdflib_file_path(
+        data_set_uri,
+        csvw_metadata_json_path,
+    )
     csv_url = select_qb_csv_url(
```

Comment:
Can you replace this sparql query call here with a call to the `data_cube_state.get_cube_identifiers_for_data_set(...)` function?

Reply:
Replaced sparql query call with access via `data_cube_state` and removed associated imports.